app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -384,6 +374,32 @@ fun ConversationScreen(
         onFailedMessageRetryClicked = messageComposerViewModel::retrySendingMessage,
         requestMentions = messageComposerViewModel::searchMembersToMention,
         onClearMentionSearchResult = messageComposerViewModel::clearMentionSearchResult,
+        onPermissionPermanentlyDenied = {
+            val (title, description) = when (it) {
+                is PermissionDenialType.CaptureVideo -> {
+                    R.string.app_permission_dialog_title to R.string.record_video_permission_dialog_description
```

Comment:
suggestion: `R.string.app_permission_dialog_title` is always the same for all permission types so it could be extracted out of this `when`